### PR TITLE
styling updates

### PR DESCRIPTION
**Pull request by howardchung**

### CODE REVIEW

File: components/Match/Match.jsx [hunk 1]
```diff
@@ -44,9 +50,38 @@ class RequestLayer extends React.Component {
...
+          <MatchTable columns={overviewColumns} />
+        </Card>
+        <Card>
+          <AbilityUpgradesTable columns={abUpgradeColumns} />
+        </Card>
+        <Card>
+          <BuildingMap match={this.props.match} loading={this.props.loading} />
+        </Card>
+        <MatchTable columns={benchmarksColumns(this.props.match)} />
       </div>
```
Comment (mike-robertson) [inline components/Match/Match.jsx]: you might want to consider doing something like the player pages components in the future once the match tabs are increased, then you can put the header in this matches component and similar to the player header.

File: components/Table/Table.jsx [hunk 2]
```diff
@@ -36,13 +37,17 @@ const getTable = (data, columns, sortState, sortField, sortClick, numRows) => {
...
+              style={{ borderBottomWidth: '0px' }}
+            >
+              {columns.map((column, colIndex) => (
+                <MaterialTableRowColumn key={colIndex} style={getWidthStyle(column.width, totalWidth)}>
+                  {row && column.displayFn && column.displayFn({ row, column, field: row[column.field] })}
+                  {row && row[column.field] && !column.displayFn && (row[column.field].display)}
+                </MaterialTableRowColumn>
+              ))}
+            </MaterialTableRow>
           ))}
```
Comment (mike-robertson) [inline components/Table/Table.jsx]: Is there a reason we are decomponentizing the table?

File: components/Table/TableRowColumn.jsx [hunk 3]
```diff
@@ -1,10 +0,0 @@
-import React from 'react';
-import { TableRowColumn as MaterialTableRowColumn } from 'material-ui/Table';
-import { getWidthStyle } from './tableHelpers';
-
-export default ({ row, column, totalWidth }) => (
-  <MaterialTableRowColumn style={getWidthStyle(column.width, totalWidth)}>
-    {row && column.displayFn && column.displayFn({ row, column, field: row[column.field] })}
-    {row && row[column.field] && !column.displayFn && (row[column.field].display)}
-  </MaterialTableRowColumn>
-);
```
Comment (mike-robertson) [inline components/Table/TableRowColumn.jsx]: any reason we are doing this? Less maintainable in my opinion to create a huge table component, and the table is concerned with layout. ONly the table row column needs to care about data.

File: components/Table/Table.jsx (hunk 2 above)
Comment (howardchung) [inline components/Table/Table.jsx]: * It was breaking the ability to pass the material-ui props down (workaround is to add them into the wrapper component, but that seems unnecessarily verbose)
* It's easier to follow what's going on

Did we need to componentize for a specific reason?

File: components/Table/TableRowColumn.jsx (hunk 3 above)
Comment (howardchung) [inline components/Table/TableRowColumn.jsx]: See the other comment

**Pull request merged**